lesson05/assignment/index.html

```diff
@@ -9,7 +9,56 @@
     </head>
     <body>
         <main class="main">
-          
+            <div class="grid ">
+                <div class="item" tabindex="1">
+                  <div class="box">
+                    <div class="header">Header</div>
+                  </div>
+                </div>
+                <div class="item" tabindex="2">
+                  <div class="box">
+                    <div class="nav_item">Nav Item</div>
+                  </div>
+                </div>
+                <div class="item" tabindex="3">
+                  <div class="box">
```

Comment:
You can add the BEM requirement to your nav by changing/adding the `box` class to `item__box` and then swap the nav item `<div>` with a `<a class="item__lin" href="#">`